testing/tests_bls12_381/tests/tests_fp/mod.rs

```diff
@@ -0,0 +1,231 @@
+use fuels::{
+    prelude::*,
+    tx::{ConsensusParameters, ContractId},
+};
+
+abigen!(BlsTestContract, "out/debug/tests_bls12_381-abi.json");
+
+//TODO move this to a separate helpers file. How to achieve this in Rust/Cargo? 
```

Comment:
Use Rust modules e.g. `helpers/mod.rs` where `helpers` is some other folder you create. Once you have enough helpers, you can further refactor that into its own modules.